addons/resource.language.en_gb/resources/strings.po

```diff
@@ -18153,3 +18153,23 @@ msgstr ""
 msgctxt "#38025"
 msgid "Choose information provider"
 msgstr ""
+
+#: system/settings/rbp.xml
+msgctxt "#38026"
+msgid "Decode the stereo stream from 3D files"
```

Comment:
any chance to get this label a bit less technical? Technical stuff can be in the description of the setting, but the main label should ideally not be to technical. So maybe something in the direction of "Process stereoscopic streams (MVC)".